pkg/hints/ec_hint.go

```diff
@@ -0,0 +1,118 @@
+package hints
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/hints/hint_utils"
+	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	"github.com/lambdaclass/cairo-vm.go/pkg/types"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+type BigInt3 struct {
+	Limbs []lambdaworks.Felt
+}
+
+type EcPoint struct {
+	X BigInt3
+	Y BigInt3
+}
+
+func (val *BigInt3) Pack86() big.Int {
+	sum := big.NewInt(0)
+	for i := 0; i < 3; i++ {
+		felt := val.Limbs[i]
+		signed := felt.ToSigned()
+		shifed := new(big.Int).Lsh(signed, uint(i*86))
+		sum.Add(sum, shifed)
+	}
+	return *sum
+}
+
+func FromBaseAddr(addr memory.Relocatable, virtual_machine vm.VirtualMachine) (BigInt3, error) {
+	limbs := make([]lambdaworks.Felt, 0)
+	for i := 0; i < 3; i++ {
+		felt, err := virtual_machine.Segments.Memory.GetFelt(addr.AddUint(uint(i)))
+		if err == nil {
+			limbs = append(limbs, felt)
+		} else {
+			return BigInt3{}, errors.New("Identifier has no member")
+		}
+	}
+	return BigInt3{Limbs: limbs}, nil
+}
+
+func FromVarName(name string, virtual_machine vm.VirtualMachine, ids_data hint_utils.IdsManager) (EcPoint, error) {
```

Comment:
```suggestion
func BigInt3FromVarName(name string, virtual_machine vm.VirtualMachine, ids_data hint_utils.IdsManager) (EcPoint, error) {
```
See previous comment